### ISSUE TITLE
Unity Editor 2021 IAsyncDisposaple Multiple dll issue

### ISSUE THREAD

**Issue by TumayVarel** [Cysharp/ZLogger#58]
In Unity Editor 2021.4.3, Zlogger gives two compile error:
```

Assets\Scripts\ZLogger\AsyncStreamLineMessageWriter.cs(12,69): error CS0433: The type 'IAsyncDisposable' exists in both 'Microsoft.Bcl.AsyncInterfaces, Version=1.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' and 'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'

Assets\Scripts\ZLogger\IAsyncLogProcessor.cs(7,43): error CS0433: The type 'IAsyncDisposable' exists in both 'Microsoft.Bcl.AsyncInterfaces, Version=1.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' and 'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'
```

**Comment by GCodergr** [comment]
You probably mean Unity 2021.3.4f1 . Got the same error too.

**Comment by SaNeOr** [comment]
> ZLogger.Shims folder

Where is it?

**Comment by neuecc** [comment]
fixed at 1.7.0.
